default/python/common/configure_logging.py

```diff
@@ -1,26 +1,124 @@
+"""
+configure_logging redirects print and the logger to use the freeorion server.
+
+Output redirected to the freeorion server prints in the appropriate log:
+freeorion.log freeoriond.log or AI_<N>.log.
+
+logging messages of levels warning and above are decorated with module name, file name, function name and line number.
+
+Usage:
+
+import logging
+import utils.configure_logging
+< Use python logging or print and have it re-directed >
+
+Notes on using the standard python logger:
+The python standard library is composed of two basic parts: loggers and handlers.
+Loggers generate the log and associate a whole bunch of level, timing, file, function and
+other information with the log.  Handlers do somthing like stream to file, console or email.
+
+* The simplest is to use the root logger.
+logging.debug(msg)
```

Comment:
Please add description to each log where it should be used.  

IMHO 3 loggers will be enough, other can be marked as not recommended.